packages/cdktf/test/assets.test.ts

```diff
@@ -0,0 +1,113 @@
+import { App, TerraformHclModule, TerraformStack, Testing } from "cdktf";
```

Comment:
These need to be imported directly from the `lib` folder.

Reply:
fixed